test/esm-node-native/package.json

```
@@ -0,0 +1,6 @@
+{
+  "type": "module",
+  "scripts": {
+    "test": "node --experimental-specifier-resolution=node index.js"
```

```suggestion
    "test": "node index.js"
```

The `--experimental-specifier-resolution` flag isn't recommended unless you really need extension resolution.

---

Thanks, this was based on the sample I was using 👍🏻